src/components/InterruptionSettings.js

```diff
@@ -0,0 +1,334 @@
+import React, { useState, useEffect, useRef } from "react";
+import IntervalSlider from "./IntervalSlider";
+import WebsiteField from "./WebsiteField";
+import Checkbox from "@material-ui/core/Checkbox";
+import FormGroup from "@material-ui/core/FormGroup";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import WebsiteForm from "./WebsiteForm";
+
+const styles = {
+  row: {
+    display: "flex",
+    flexDirection: "row",
+    alignItems: "center",
+    justifyContent: "space-between",
+  },
+};
+
+export const InterruptionSettings = (props) => {
+  const [interval, setInterval] = useState(15);
+  const [sharedState, setSharedState] = useState(props.websitesObject);
+  const [allDisabled, setAllDisabled] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+  const [urlInput, setURLInput] = useState("https://");
+  const [nameInput, setNameInput] = useState("");
+  const [isClicked, setIsClicked] = useState(false);
+
+  useEffect(() => {
+    changingInterval();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  const changingInterval = () => {
```

Comment:
you know how the romans used to say 'de gustibus non disputandum' 

but... isn't `function changingInterval() {} ` easier to read than `const changingInterval = () => {}` ?